java/src/com/ibm/streamsx/topology/generator/spl/OperatorGenerator.java

```diff
@@ -93,8 +93,12 @@ private void parallelAnnotation(JSONObject op, StringBuilder sb) {
         if (parallel != null && parallel) {
             sb.append("@parallel(width=");
             Object width = op.get("width");
-            if (width instanceof Integer)
+            if (width instanceof Integer) {
                 sb.append(Integer.toString((int) width));
+        	}
```

Comment:
JSON4J reads any integral value as a long from serialized JSON.

However, parallel width is an integer, so the Java api should continue to be an int/Integer.

When reading the JSON, just do intValue() on the returned Number